pyproject.toml

```diff
@@ -13,6 +13,7 @@ testpaths = "tests"
 filterwarnings = [
   "error",
   "ignore::UserWarning",
+  "ignore::PendingDeprecationWarning",
```

Comment:
Really?

Reply:
I don't know how to make this specific to Matplotlib. This arose when updating to Matplotlib 3.6 where `get_cmap` raises this warning. They say to use the syntax
```Py
cmap = matplotlib.colormaps[name]
```
instead of
```Py
cmap = matplotlib.cm.get_cmap(name)
```
However, the new syntax does not exist in `mpl<3.5`. I thought I would keep it compatible?
Alternatively I can just use the new syntax and put a lower bound on the MPL requirement?
What do you think?

Reply:
Leave it for now.